staging/src/k8s.io/client-go/examples/third-party-resources/types.go

```diff
@@ -23,6 +23,18 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/schema"
 )
 
+const (
+	ExampleDomain              = "k8s.io"
```

Comment:
This domain was there before, I just moved it to constant, but I agree.

> some sub-domain?

Do you mean something like "tpr.k8s.io" or completely different domain ("example.com" would clash with "example" resource there)? Any suggestions?